molo/polls/tests/test_voting.py

```diff
@@ -50,6 +52,46 @@ def test_voting_once_only(self):
             kwargs={'poll_id': question.id}))
         self.assertContains(response, '100%')
 
+    def test_multiple_options_with_translations(self):
```

Comment:
The name of the test seems incorrect to me? the choices you've created don't have translations